### PR TITLE
Documentation for Local NI with multiple ports

### PR DESCRIPTION
**Pull request by milan-zededa**
Added documentation for recently merged support for Local NI with multiple ports.

### CODE REVIEW

**Review by rucoder — APPROVED** [review]
test

File: docs/NETWORKING.md [hunk 1]
```diff
@@ -93,8 +99,9 @@ Application then references UUIDs of network instances to which it should be con
 
 ### Uplink
 
-Uplink denotes network adapter that a network instance is connected to for external connectivity.
-Network instance can be air-gapped, i.e. without uplink, thus not providing external connectivity.
+Uplink denotes network adapter used for EVE management, i.e. providing connectivity
+with the controller. Multiple adapters can be used for management and EVE will automatically
+fail-over between them when some loose connectivity.
```
Comment (naiming-zededa) [inline docs/NETWORKING.md:104]: probably should be 'lost connectivity', loose seems to be not connected very tight.

**Review by naiming-zededa — COMMENTED** [review]
LGTM

File: docs/APP-CONNECTIVITY.md [hunk 2]
```diff
@@ -229,62 +231,106 @@ are still supported and must be configured to allow anything beyond just DNS and
 Since there is no NAT between applications and external endpoints, properly configured
 inbound ACL rules are that much more important.
 
-Metadata HTTP server is run for switch network instance only if it has uplink port with
-an IP address.
+A metadata HTTP server is run for a switch network instance only if it has a port attached
```
Comment (eriknordmark) [inline docs/APP-CONNECTIVITY.md:234]: Was this correct before your change? I didn't think we need a (external/uplink) port, but that we do need an bridge IP address assigned to EVE, meaning that it needs to be a local NI.
If it needs to be fixed a separate PR (or commit) would make sense.

**Review by eriknordmark — APPROVED** [review]
LGTM

File: docs/APP-CONNECTIVITY.md (hunk 2 above)
Comment (milan-zededa) [reply docs/APP-CONNECTIVITY.md:234]: For local network instance we indeed do not need port to start metadata server.
But for switch NI (which is described in this doc section), we start it only if we get IP address on the bridge. This happens only if NI has a (single) port attached and that port receives an IP address (statically or via DHCP, i.e. it is not set with "DHCP passthrough"). Then we install ACL to prevent access to that metadata server from outside.
